File: repos_estudiantes_ALVAREZ_segunda_entrega/GUTIERREZ_GARCIA_CARLA_ALEJANDRA/COMPOSICION_AGREGACION/Ejercicio5/Jugador.java
package Ejercicio5;

public class Jugador {
    private String nombre;
    private int numero;
    private String posicion;

    public Jugador(String n, int num, String pos) {
        this.nombre = n;
        this.numero = num;
        this.posicion = pos;
    }

    public void mostrar_info() {
        System.out.println("Nombre: " + this.nombre + ", Numero: " + this.numero + ", Posicion: " + this.posicion);
    }

    public String getNombre() {
        return nombre;
    }

    public int getNumero() {
        return numero;
    }

    public String getPosicion() {
        return posicion;
    }

    public void setNombre(String n) {
        this.nombre = n;
    }

    public void setNumero(int num) {
        this.numero = num;
    }

    public void setPosicion(String pos) {
        this.posicion = pos;
    }
}

File: repos_estudiantes_TARQUI_segunda_entrega/RUBIN_DE_CELIS_VALDEZ_ANDRES/COMPOSICION/EJERCICIO-5/Jugador.py
class Jugador:
    def __init__(self, nombre, numero, posicion):
        self.nombre = nombre
        self.numero = numero
        self.posicion = posicion

    def get_nombre(self):
        return self.nombre

    def get_numero(self):
        return self.numero

    def get_posicion(self):
        return self.posicion

    def set_nombre(self, nombre):
        self.nombre = nombre

    def set_numero(self, numero):
        self.numero = numero

    def set_posicion(self, posicion):
        self.posicion = posicion

    def mostrar_info(self):
        print(f"Nombre: {self.nombre}, Numero: {self.numero}, Posicion: {self.posicion}")